### DISCUSSION TITLE
下载需要会员视频的弹幕

### DISCUSSION THREAD

**Discussion by ShuiHuo** [the1812/Bilibili-Evolved#2451] [功能建议 / Ideas]
视频链接 https://www.bilibili.com/bangumi/play/ep426335 进入功能页面后没看到下载弹幕之类的功能，点击 转到BiliPlus 里面的页面虽然有下载弹幕，但麻烦了点，希望能给这种会员视频也添加直接下载弹幕的按钮。

顺便一提，升到 2.0.0 后进入功能页面太难了，鼠标在页面左边找半天都找不到位置（左边还有一个屏幕，鼠标经常划过头飘出去）。

**Comment by the1812** [comment]
> 点击 转到BiliPlus 里面的页面虽然有下载弹幕，但麻烦了点，希望能给这种会员视频也添加直接下载弹幕的按钮。

那是 BiliPlus 提供的下载弹幕功能, 想直接在播放页下载得开大会员.

> 顺便一提，升到 2.0.0 后进入功能页面太难了

难在哪了, 位置不是和旧版一样的吗

**Reply by ShuiHuo** [reply]
旧版的按钮突出，能直接点击。![2021年11月01号_13点19分59秒_grim](https://user-images.githubusercontent.com/7471938/139625165-c5f0fa6f-7308-4c45-887a-265547d4eae9.png)

新版完全隐藏了，鼠标没到边上缝隙那一小块地方就不会将按钮弹出来。

**Reply by the1812** [reply]
你是装了`自动隐藏侧栏`?

**Reply by ShuiHuo** [reply]
还真是这个导致的。更新到 2.0.0 后直接用了 `我全都要`，没想到这点😂
